examples/huggingface/pytorch/text-generation/quantization/llm_quantization_recipes.md

```diff
@@ -6,15 +6,25 @@ The scripts [run_generation.py](./run_generation.py) provide two quantization ap
 
 # Validated Models
 
-|                       Model Name                       |
-| :----------------------------------------------------: |
-|       [EleutherAI/gpt-j-6b](#eleutheraigpt-j-6b)       |
-|         [facebook/opt-1.3b](#facebookopt-13b)          |
-|          [facebook/opt-30b](#facebookopt-30b)          |
-|  [meta-llama/Llama-2-7b-hf](#meta-llamallama-2-7b-hf)  |
-| [meta-llama/Llama-2-13b-hf](#meta-llamallama-2-13b-hf) |
-| [meta-llama/Llama-2-70b-hf](#meta-llamallama-2-70b-hf) |
-|         [tiiuae/falcon-40b](#tiiuaefalcon-40b)         |
+|                             Model Name                              |
+| :-----------------------------------------------------------------: |
+|             [EleutherAI/gpt-j-6b](#eleutheraigpt-j-6b)              |
+|                [facebook/opt-1.3b](#facebookopt-13b)                |
+|                [facebook/opt-30b](#facebookopt-30b)                 |
+|        [meta-llama/Llama-2-7b-hf](#meta-llamallama-2-7b-hf)         |
+|       [meta-llama/Llama-2-13b-hf](#meta-llamallama-2-13b-hf)        |
+|       [meta-llama/Llama-2-70b-hf](#meta-llamallama-2-70b-hf)        |
+|               [tiiuae/falcon-40b](#tiiuaefalcon-40b)                |
+|                [tiiuae/falcon-7b](#tiiuaefalcon-7b)                 |
+|  [baichuan-inc/Baichuan2-7B-Chat](#baichuan-inc/Baichuan2-7B-Chat)  |
```

Comment:
Do you mark specific transformers==4.33.3 for this model?

Reply:
> Do you mark specific transformers==4.33.3 for this model?

These models already support transformers==4.35.2